cdap-api/src/main/java/co/cask/cdap/api/dataset/table/RowColumns.java

```diff
@@ -43,16 +48,19 @@
 
   public RowColumns(byte[] row) {
     this.row = row;
-    this.columns = new ArrayList<>();
   }
 
   public RowColumns(byte[] row, byte[]... columns) {
     this(row);
```

Comment:
Better not to call `this`, since the idea is to have this constructor represents some columns (potentially zero), while the other constructor is for all columns.